toolboxTreppe/toolboxTreppe.ino

```diff
@@ -106,9 +103,10 @@ void animation1(uint8_t& y, uint8_t& x, uint8_t& gegenstrecke) {
 }
 //----------------------------- Debug Funktion ----------------
 
-void debugPrint(const char* output[]) {
+void debugPrint(const char* output[], ...) {
```

Comment:
void debugPrint(const char* output, ...);

Reply:
already changed on thursday after we talked but not pushed yet